SPECS/kernel/kernel.signatures.json

```diff
@@ -1,7 +1,7 @@
 {
  "Signatures": {
-  "config": "b8c9e2a875e4e6655fdbeb626088529fd1cef401b8f67a481fc301d2a3a026c5",
+  "config": "681e8ff274fc3fc79eaf38d5152e99702e22346bd2aaff06925715dcf1a00fc3",
   "config_aarch64": "3057cf5c5f04b57c4d69f9783d4809de217fb46a4278694c19f6c3ffd81249c5",
```

Comment:
Are there no changes to the aarch64 config?

Reply:
Good catch! Checking to see what happened here...

Reply:
Fixed